### DISCUSSION TITLE
Discussion / Question: Is it possible to gain access to the Paragon Library Repository?

### DISCUSSION THREAD

**Discussion by chronosacaria** [malcolmriley/minecraft-textiles#50] [Ideas and Suggestions]
Hi @malcolmriley! I hope this message finds you well! I have started on the port of Minecraft Textiles and I realised that I cannot grab the Paragon Library Repo as it either no longer exists or is not public. Is there any way that you could either invite me to it or make it public long enough for me to grab it for my project? Thank you, so much!

I hope that you have a wonderful day!

**Comment by malcolmriley** [comment]
I originally made it private as it really isn't suitable for consumption as an API. It's poorly commented and I make breaking changes whenever I need to. As you can no doubt see I simply tend to use it as a submodule rather than as a properly built library.

I've made it public (MIT license) and disclosed its shortcomings since there really isn't any harm in letting people take a look, and obviously those such as yourself have a legitimate interest in using it.

Thanks again for porting, and I hope your day is likewise wonderful!

**Reply by chronosacaria** [reply]
Thank you so much. Sorry, I didn't realise that there was the Discussions section. I don't know why I made an issue. :P

**Reply by malcolmriley** [reply]
You're welcome and no worries about the Issue/Discussion thing, truly. I didn't mention it because it wasn't a problem. Happy modding!

**Comment by malcolmriley** [comment]
Hello again, @chronosacaria! Wasn't quite sure how to contact you so I hope you receive a notification for this message.

I just updated WilyTextiles to 1.18.2 and as a part of that process updated the obfuscation mappings to the official (Mojang) channel. As I understand it, [it is possible to use these mappings within Fabric](https://fabricmc.net/wiki/tutorial:mappings). Hopefully this reduces the number of obstacles you might be facing with your potential port.

**Reply by chronosacaria** [reply]
That's awesome! It would definitely help in terms of making the porting process easier. I use Yarn Mappings, though, because of copyright concerns associated with Mojmaps, but I can read the Mojmaps a lot easier than MCP lol. Thanks, so much, for letting me know! :)
